cmd/snap-confine/snap-confine.c

```diff
@@ -112,6 +112,10 @@ int main(int argc, char **argv)
 #endif				// ifdef HAVE_SECCOMP
 
 	if (geteuid() == 0) {
+		// ensure that "/" or "/snap" is mounted with the
```

Comment:
If you don't lock it may get nasty. I'd rather move the locking elsewhere and allow us to do some host setup while holding a lock than to do this incorrectly.